package.json

```diff
@@ -63,9 +64,12 @@
   "dependencies": {
     "arr-flatten": "^1.0.1",
     "ava-init": "^0.1.0",
-    "babel-core": "^5.8.23",
-    "babel-plugin-espower": "^1.1.0",
-    "babel-runtime": "^5.8.29",
+    "babel-core": "^6.1.20",
+    "babel-plugin-espower": "^2.0.0",
+    "babel-plugin-transform-runtime": "^6.1.18",
+    "babel-preset-es2015": "6.0.14",
+    "babel-preset-stage-3": "6.0.14",
+    "babel-runtime": "^5.8.34",
```

Comment:
> the reason I need to depend on babel-runtime@5 here is because of the way we are mucking with module paths

This may not be true. I just discovered that running the babel test suite had created a whole bunch of global symlinks. Not sure if it was what was causing this, I will retest later.